update INFO doc for current_fork_perc, current_save_keys_processed, c… (#1518)
redis/redis-doc: commands/info.md

*   `loading`: Flag indicating if the load of a dump file is on-going
*   `current_cow_size`: The size in bytes of copy-on-write memory
     while a child fork is running

> *   `current_fork_perc`: The progress in percentage of the current fork process, in AOF or RDB fork it indicates the percentage of processed keys

Reviewer: ```suggestion
*   `current_fork_perc`: The percentage of progress of the current fork process. For AOF and RDB forks it is the percentage of `current_save_keys_processed` out of `current_save_keys_total`.
```
